packages/tools/src/utilities/scroll.ts

```diff
@@ -59,4 +61,32 @@ export function scrollVolume(
     position: newPosition,
   });
   viewport.render();
+
+  const { numScrollSteps, currentStepIndex } =
+    csUtils.getVolumeViewportScrollInfo(viewport, volumeId);
```

Comment:
I"m a little bit confused, isn't it the fact that this function relies on the `sliceRangeInfo` that is calculated above? So can we refactor its logic so that we don't do `sliceRangeInfo` getter again?

```
function getVolumeViewportScrollInfo(
  viewport: IVolumeViewport,
  volumeId: string
) {
  const { sliceRange, spacingInNormalDirection } = getVolumeSliceRangeInfo(
    viewport,
    volumeId
  );

  const { min, max, current } = sliceRange;

  // Now we can see how many steps there are in this direction
  const numScrollSteps = Math.round((max - min) / spacingInNormalDirection);

  // Find out current frameIndex
  const fraction = (current - min) / (max - min);
  const floatingStepNumber = fraction * numScrollSteps;
  const currentStepIndex = Math.round(floatingStepNumber);

  return { numScrollSteps, currentStepIndex };
}
```

to 

```
function getVolumeViewportScrollInfoFromSliceRange(
  sliceRange,
  spacingInNormalDirection
) {
  const { min, max, current } = sliceRange;

  // Now we can see how many steps there are in this direction
  const numScrollSteps = Math.round((max - min) / spacingInNormalDirection);

  // Find out current frameIndex
  const fraction = (current - min) / (max - min);
  const floatingStepNumber = fraction * numScrollSteps;
  const currentStepIndex = Math.round(floatingStepNumber);

  return { numScrollSteps, currentStepIndex };
}
```

Reply:
API check failed

Reply:
Ah yeah I need to build updateapi